### DISCUSSION TITLE
Real-world application benchmarks

### DISCUSSION THREAD

**Discussion by bcluap** [quarkusio/quarkus#37375] [Community]
Just posting here in case anyone is not aware of this:

https://medium.com/deno-the-complete-reference/list-of-all-of-my-real-world-performance-comparisons-a8e9182ac50

I find the comparisons really interesting. The ones where Quarkus is benchmarked:

https://medium.com/deno-the-complete-reference/url-shortener-service-benchmarking-node-express-cluster-vs-quarkus-7c0497b78bd5
https://medium.com/deno-the-complete-reference/url-shortener-service-benchmarking-node-fastify-cluster-vs-quarkus-cf3e59ec6a3a
https://medium.com/deno-the-complete-reference/url-shortener-service-benchmarking-deno-vs-quarkus-a74e2cb47867
https://medium.com/deno-the-complete-reference/url-shortener-service-benchmarking-bun-vs-quarkus-067f878bd7cc
https://medium.com/deno-the-complete-reference/url-shortener-service-benchmarking-go-gin-vs-quarkus-ee22b55ed64c
https://medium.com/deno-the-complete-reference/url-shortener-service-benchmarking-spring-boot-mvc-vs-quarkus-6a47c3136741
https://medium.com/deno-the-complete-reference/url-shortener-service-benchmarking-quarkus-vs-rust-actix-606d46fff88b

**Comment by mkouba** [comment]
CC @franz1981

**Comment by franz1981** [comment]
Thanks, yep we send few comments in many of this, sadly, not very scientific nor right/fair benchmarks :"(
Reading comments in medium is never easy but you can find me and others quarkus folks in few of them
